src/lib/y2network/config_reader/sysconfig.rb

```diff
@@ -33,9 +33,14 @@ class Sysconfig
       def config
         interfaces = find_interfaces
         # load /etc/sysconfig/network/routes
-        routing = Y2Network::Routing.new(tables: [SysconfigRoutesReader.new.config])
-        # TODO: SysconfigRoutesReader(s) for ifroute-* files
-        Config.new(interfaces: interfaces, routing: routing, source: :sysconfig)
+        routes = SysconfigRoutesReader.new.config
+        # load /etc/sysconfig/network/ifroute-*
+        dev_routes = interfaces.map do |iface|
+          SysconfigRoutesReader.new(routes_file: "/etc/sysconfig/network/ifroute-#{iface.name}").config
+        end
+        routing = dev_routes.inject(routes) { |memo, r| memo.concat(r.routes) }.uniq
```

Comment:
I would not call this variable `routing` as it might be confused with a `Y2Network::Routing` instance.

Reply:
Additionally, I would probably move this logic to a `find_routes` (sure you can come up with a better name) method to hide the complexity a bit (analogous to the `find_interfaces` one). Tomorrow we will need to read DNS info, etc. and this module will grow pretty quickly.